cycle_en_terre_website_design/static/src/less/cycle_en_terre_website_design.less

```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+@import url('https://fonts.googleapis.com/css?family=Old+Standard+TT');
+
+body {
+    font-size: 16px;
+    line-height: 25px;
+    font-weight: 500;
+    letter-spacing: 0px;
+    color: #626262;
+    font-family: Old Standard TT, Arial, Tahoma, sans-serif;
+}
+
+a {
+    color: #f0b61c;
+}
+
+a:hover {
+    text-decoration:underline;
+    color: #a67c0a;
+}
+
+a, a:visited, a:focus {
+    text-decoration: none;
+    outline: 0;
+    }
+
+#wrap {
+   padding-top:60px;
+   padding-bottom:80px;
+}
+
```

Comment:
All the following properties changing the `.navbar` (and related classes) must be more precise. You don't want to change all the navbars but only the one in the header part of the pages.

To fix this simply put `header` before all the `.navbar` properties. Example : 
```
header .navbar-default {
    background-color:inherit;
}

header .navbar-right {
    float: none;
    margin-right: 0;
}
…
```

Reply:
Done